assets/javascripts/discourse-assign/connectors/groups-interaction-custom-options/assignable-interaction-fields.hbs

```diff
@@ -11,7 +11,7 @@
     @valueProperty="value"
     @value={{assignableLevel}}
     @content={{assignableLevelOptions}}
-    @class="groups-form-assignable-level"
+    class="groups-form-assignable-level"
```

Comment:
I don't this works. d-button has explicitly added `...attributes` to its element, but select-kit's combo box doesn't have that

Reply:
For classic components, attributes are added to the wrapper element (assuming `tagName` is not overridden to `""`)